src/init.cpp

```diff
@@ -1029,8 +1079,12 @@ bool AppInit2(boost::thread_group& threadGroup)
                 pcoinscatcher = new CCoinsViewErrorCatcher(pcoinsdbview);
                 pcoinsTip = new CCoinsViewCache(pcoinscatcher);
 
-                if (fReindex)
+                if (fReindex) {
                     pblocktree->WriteReindexing(true);
+                    //If we're reindexing in prune mode, wipe away all our block and undo data files
```

Comment:
We can do that, but we didn't really change the behavior of reindex.  All we did was delete the old data files which wouldn't have been found or used anyway and would just have gotten in the way of the new files being written.  

Sent from my iPhone

> On Apr 17, 2015, at 8:02 AM, Jonas Schnelli <notifications@github.com> wrote:
> 
> In src/init.cpp:
> 
> >                      pblocktree->WriteReindexing(true);
> > +                    //If we're reindexing in prune mode, wipe away all our block and undo data files
> Agreed. I just wiped my 6h sync by trying to see what -reindex does in pruned mode. :)
> 
> —
> Reply to this email directly or view it on GitHub.
>